lua/code_action_menu/lsp_objects/actions/code_action.lua

```diff
@@ -80,7 +80,24 @@ function CodeAction:execute()
   if self:is_workspace_edit() then
     vim.lsp.util.apply_workspace_edit(self.server_data.edit, 'utf-8')
   elseif self:is_command() then
-    vim.lsp.buf.execute_command(self.server_data.command)
+    local client = vim.lsp.get_client_by_id(self.client_id)
+    local fn = client.commands[self.server_data.command.command]
+      or vim.lsp.commands[self.server_data.command.command]
+    if fn then
+      local context = {}
+      context.diagnostic = vim.lsp.diagnostic.get_line_diagnostics()
+      local params = vim.lsp.util.make_range_params()
+      params.context = context
+
+      fn(self.server_data.command, {
```

Comment:
Related to the above comment: we just need to pass `self.server_data` here. The `self.server_data.command` is just the `string`.

Reply:
Nope, `self.sever_data.command` is a table.

Add the following at line 91

```lua
print(vim.inspect('here --> ' .. type(self.server_data.command)))
```

See the output https://github.com/weilbith/nvim-code-action-menu/issues/49#issue-1185397324